src/main/java/de/dafuqs/spectrum/registries/SpectrumItemStackDamageImmunities.java

```diff
@@ -22,6 +22,11 @@ public static void registerDefaultItemStackImmunities() {
 		SpectrumItemStackDamageImmunities.addImmunity(SpectrumBlocks.WET_LAVA_SPONGE.asItem(), DamageTypeTags.IS_FIRE);
 		SpectrumItemStackDamageImmunities.addImmunity(SpectrumBlocks.DOOMBLOOM.asItem(), DamageTypeTags.IS_FIRE);
 		SpectrumItemStackDamageImmunities.addImmunity(SpectrumItems.DOOMBLOOM_SEED, DamageTypeTags.IS_FIRE);
+
+		SpectrumItemStackDamageImmunities.addImmunity(SpectrumItems.PURE_NETHERITE_SCRAP.asItem(), DamageTypeTags.IS_FIRE);
+		SpectrumItemStackDamageImmunities.addImmunity(SpectrumBlocks.PURE_NETHERITE_SCRAP_BLOCK.asItem(), DamageTypeTags.IS_FIRE);
+		SpectrumItemStackDamageImmunities.addImmunity(SpectrumItems.PURE_NETHERITE_SCRAP.asItem(), SpectrumDamageTypeTags.IS_PRIMORDIAL_FIRE);
```

Comment:
Primordial Fire already is in the `IS_FIRE` tag, so the `IS_PRIMORDIAL_FIRE` tags won't add anything to it, but it also won't break anything. Can be removed (low prio)

Reply:
done.